regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -37,6 +40,7 @@ var MainView = Backbone.View.extend({
             this.externalEvents.on('breakaway:open', this.breakawayOpen, this);
             this.externalEvents.on('section:error', this.displayError, this);
         }
+        this.externalEvents.on('section:resize', this.redrawTables, this);
```

Comment:
If this were `this. applyTablePlugin`, we wouldn't need the `redrawTables` function, hey?